CRM/PivotCache/GroupCase.php

```diff
@@ -17,7 +17,7 @@ public function query($page, array $params) {
 
     $cache->group_name = $this->getName();
 
-    $cache->whereAdd("path <> 'header'");
+    $cache->whereAdd("path NOT IN ('header', 'entityCount', 'pivotCount')");
```

Comment:
Would I be wrong in believing we can move this whole function to CRM_PivotCache_AbstractGroup? I mean, you had to implement the same exact change on 4 different files.

Reply:
Yeah, good point, I'll change it.